CHANGELOG.md

```diff
@@ -8,11 +8,24 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### ⚠️ Notice ⚠️
```

Comment:
This shouldn't be it's own section.

What about [alert syntax](https://github.blog/changelog/2023-12-14-new-markdown-extension-alerts-provide-distinctive-styling-for-significant-content/)?

Reply:
I have moved it to the deprecated section with a warning.